client/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hp-admin-crypto-client"
```

Comment:
TBH I'd rather standardize on `Keypair` so that we're consistent with `ed25519_dalek`. Also, scope is set using `--scope` flag rather than here I think. Additionally, this might end up including more than just Keypair wrapper.

Reply:
I like that name. Accepted.